api/v1beta1/disk_failure.go

```diff
@@ -18,6 +18,50 @@ type OpenatSyscallSpec struct {
 	ExitCode string `json:"exitCode"`
 }
 
+// DiskFailureSpec represents a disk failure disruption
+type DiskFailureSpec struct {
+	// +kubebuilder:validation:Required
+	// +ddmark:validation:Required=true
+	Paths []string `json:"paths"`
```

Comment:
Are we ok to just drop a field?
It seems it's a breaking change for me 🤔 

Maybe it's ok if we did not released this as part of a numbered release already 🤷